Sources/SwiftCliCore/Errors.swift

```diff
@@ -6,3 +6,7 @@ enum PatternValidationError: Error {
     case noStitches
     case invalidStitchFound
 }
+
+enum DeveloperError: Error {
```

Comment:
Unclear to me what `DeveloperError` entails.

Reply:
This was going to be an error thrown if a function was called with inappropriate input, but then I abandoned it and never used it anywhere. Maybe I should just delete these until I'm going to use them?

Reply:
Yes, usually in a PR, you want to stick to just committing and pushing the changes that are going to be used. You can move those "in progress" changes to a separate branch.